### PR TITLE
DOCS-3609: Document FCO shipping options request

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    {
+      "name":"Seur - Spain",
+      "price":7,
+      "allowed_areas":[
+        "NL",
+        "ES"
+      ]
+    },
+    {
+      "name":"TNT - verzending BE en FR",
```
Comment (JorgeSariego-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:176]: @FrancescaNCook 
We added the example on the Jira ticket ... but for documentation, I think might need to be set properly (same as others)
`"name":"TNT - verzending BE en FR",`

````
"excluded_areas":[
        "NL",
        "FR",
        "ES"
      ]
````
actually I think should be
allowed_areas BR and FR

Maybe titles could be good also in english.

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+```json
+{
+  "order_id":"apitool_8515133",
+  "amount":"9810",
+  "currency":"EUR",
+  "customer":{
+    "locale":"en_US",
+    "ip_address":"127.0.0.1",
+    "forwarded_ip":"127.0.0.1",
+    "first_name":"Test namexx",
```
Comment (JorgeSariego-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:35]: @FrancescaNCook , I think content in the JSON should be reviewed and make all in english?
first_name":"Test namexx", is for just DEV testing

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+# Integration
+
+To trigger the shipping options request, in your create order request, set `checkout_options.use_shipping_notification` (boolean) to `true`.
+
+See Recipe – [Create a FastCheckout page](/recipes/create-a-fastcheckout-page) > Step 6. Configure shipping options. 
+
+# Request
+
+The method of the shipping options request depends on whether you set `payment_options.notification_method` to `POST` (recommended) or `GET`.
```
Comment (Sarah-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:20]:
```suggestion
The HTTP method of the shipping options request depends on whether you set `payment_options.notification_method` to `POST` (recommended) or `GET` when you create an order. 
```

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+See Recipe – [Create a FastCheckout page](/recipes/create-a-fastcheckout-page) > Step 6. Configure shipping options. 
+
+# Request
+
+The method of the shipping options request depends on whether you set `payment_options.notification_method` to `POST` (recommended) or `GET`. 
+
+See Recipe – [Create a FastCheckout page](/recipes/create-a-fastcheckout-page) > Step 1. Create an order. 
+
+## POST
+
```
Comment (Sarah-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:25]: Is this an example POST request from MultiSafepay to the merchant? If so, I think we should add a sentence explaining that to help the reader.

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+| `currency` | The currency of the payment. <br> Format: <a href="https://en.wikipedia.org/wiki/ISO_4217" target="_blank">ISO-4217 currency code</a> <i class="fa fa-external-link" style="font-size:12px;color:#8b929e"></i>. |
+| `customer` | object | See Recipes – [customer object](/recipes/customer-object-1). |
+| `delivery` | object | See Recipes – [delivery object](/recipes/delivery-object). |
+| `shopping_cart` | object | See Recipes – [Display shopping cart](/recipes/display-shopping-cart). |
+
+</details>
+
+## GET
+
+The data is sent as query parameters.
```
Comment (Sarah-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:111]: Is this an example GET request from MultiSafepay to the merchant? If so, I think we should add a sentence explaining that to help the reader.

File: content/checkout/fastcheckout-shipping-options-request.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+| `amount` | The total amount of the order. |
+| `total_incl_vat` | The total amount of the order including VAT. |
+| `total_incl_vat_excl_shipping` | The total amount of the order, including VAT and excluding the shipping cost. |
+| `items_count` | The total number of items in the order. |
+
+</details>
+
+# Response
+
+We expect to receive the response formatted as follows:
```
Comment (Sarah-MultiSafepay) [inline content/checkout/fastcheckout-shipping-options-request.md:149]:
```suggestion
We expect to receive your response formatted as follows:
```

File: content/checkout/fastcheckout-shipping-options-request.md (hunk 4 above)
Comment (FrancescaNCook) [reply content/checkout/fastcheckout-shipping-options-request.md:25]: @Sarah-MultiSafepay  Good idea. Done!

**Pull request merged**